test/runnable/test16096.sh

```diff
@@ -4,23 +4,13 @@ set -e
 
 src=runnable/extra-files
 dir=${RESULTS_DIR}/runnable
-output_file=${dir}/test16096.sh.out
 
 if [ "$OS" != 'osx' ] || [ "$MODEL" != '64' ]; then
     exit 0
 fi
 
-function teardown {
-    if [ "$?" -ne 0 ] && [ -r "$output_file" ]; then
-        cat "${output_file}" 1>&2
-    fi
-    rm -f "${output_file}"
-}
```

Comment:
No need for custom rolled error handling anymore. This is the sane default behavior.

Reply:
Depending on how the makefile is written, `.DELETE_ON_ERROR` might be required.

Reply:
Oh this is now handled by the `sh_do_test` wrapper